Typescript annotation in Vue 3 with Vite

## Prerequisites

* Version: `"lucide-vue-next": "^0.16.29",`
* Operating system: `node v16.14.0`, `"typescript": "^4.4.3",`

## Step to reproduce

1. Create [a Vite project with the Vue 3 template](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).
2. Install the Vue 3 lucide package (`npm install lucide-vue-next`).
3. Use an icon somewhere (with or without the setup function).
4. Build the app (`npm run build`) <- **this fails**.

### Actual behavior

## Any message or error

```
node_modules/lucide-vue-next/dist/lucide-vue-next.d.ts:548:48 - error TS2304: Cannot find name 'FunctionalComponent'.
export declare const SomeIcon: (props: SVGProps) => FunctionalComponent<SVGProps>;
                                                   ~~~~~~~~~~~~~~~~~~~
```

## Resources

No resources.

I tried to manually import `FunctionalComponent` from vue in `lucide-vue-next.d.ts`. Afterwards the following error occurs:
```
node_modules/lucide-vue-next/dist/lucide-vue-next.d.ts: - error TS0000: Cannot find name 'FunctionalComponent'.
export declare const SomeIcon: (props: SVGProps) => FunctionalComponent<SVGProps>;
                                                       ~~~~~~~~~~~~~~~~~~~
```

--- a/packages/lucide-vue-next/scripts/buildTypes.js
+++ b/packages/lucide-vue-next/scripts/buildTypes.js
@@ -8,7 +8,7 @@ const TYPES_FILE_NAME = 'lucide-vue-next.d.ts';
 
 // Generates header of d.ts file include some types and functions
 const typeDefinitions = `\
-import { SVGAttributes, VNode } from 'vue';
+import { SVGAttributes, FunctionalComponent } from 'vue';
 declare module 'lucide-vue-next'
 
 // Create interface extending SVGAttributes

